src/components/templates/TalkShowStudio/components/Player/Player.tsx

```diff
@@ -86,7 +87,7 @@ const Player: React.FC<VideoPlayerProps> = ({
             </div>
 
             <div
-              className={IconContainerClasses(!isCamOn)}
+              className={iconContainerClasses(!isCamOn)}
```

Comment:
We shouldn't have to add the `!` here. The helper should handle any negations it may need inside itself. The calling code shouldn't have to care.